frame/auction.ts

```diff
@@ -39,3 +62,48 @@ function randomToken() {
     )
     .join("");
 }
+
+async function fetchAndValidateTrustedScoringSignals(
+  baseUrl: string,
+  hostname: string,
+  renderingUrls: ReadonlySet<string>
+) {
+  const url = new URL(baseUrl);
+  url.searchParams.append("hostname", hostname);
+  url.searchParams.append(
+    "keys",
+    [...renderingUrls].map(encodeURIComponent).join(",")
+  );
+  const response = await tryFetchJson(url.href);
```

Comment:
This will not include credentials because it is cross origin, right?

(not including them is the behavior we want)

Reply:
Thanks for pointing this out. It will generally be cross-origin, although it could be same-origin if someone self-hosts the frame. I've now added tests that make this explicit, and in #62 am making it explicit in the implementation and closing the same-origin edge case.